src/modules/cpu/host_tensor_augmentations.hpp

```diff
@@ -223,57 +251,196 @@ RppStatus brightness_u8_u8_host_tensor(Rpp8u *srcPtr,
                 dstPtrRow += dstDescPtr->strides.hStride;
             }
         }
+        // Brightness without fused output-layout toggle (NHWC -> NHWC)
```

Comment:
Can we retain the previous brightness execution style rather than separating them out for these cases?